src/libs/tiny_ttf/lv_tiny_ttf.c

```diff
@@ -278,8 +297,17 @@ static const void * ttf_get_glyph_bitmap_cb(lv_font_glyph_dsc_t * g_dsc, lv_draw
     };
 
     lv_cache_entry_t * entry = lv_cache_acquire_or_create(dsc->draw_data_cache, &search_key, (void *)font->dsc);
-
     if(entry == NULL) {
+        if(!dsc->cache_size) {  /* no cache, do everything directly */
+            if(tiny_ttf_draw_data_cache_create_cb(&search_key, (void *)font->dsc)) {
+                /* use the cache entry to store the buffer if no cache specified */
+                g_dsc->entry = (lv_cache_entry_t *)search_key.draw_buf;
+                return g_dsc->entry;
```

Comment:
Looks good! Thanks! Just one thing. Did you mean to return this? It looks like `g_dsc->entry` is of type `lv_cache_entry_t *` while a draw buf is what gets normally returned below, in the case where the cache is enabled.

Reply:
Yes, it's a deliberate recast to reuse the cache entry in non cache environment to store the draw buffer so it can be deleted in the call back - the callback always checks if cache is disabled so should only directly delete the draw buffer in correct situation.